ocs_ci/ocs/ocp.py

```diff
@@ -241,6 +241,24 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
         return self.exec_oc_cmd(command)
 
+    def patch(self, resource_name, params, type='json'):
```

Comment:
Didn't comment this method as you pointed that it's taken from other PR but as it looks this one will be merged soon I have some comments also for this method.

Reply:
ack